### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by mhowlett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,27 @@
-Confluent's Apache Kafka client for Python
-==========================================
+Confluent's Python Client for Apache Kafka<sup>TM</sup>
```
Comment (edenhill) [inline README.md]: client, not Client.

Comment (mhowlett) [inline README.md]: in fact this should be capitalized (it's a heading) and words in other headings on this page should also be capitalized.

Comment (mhowlett) [inline README.md]: rule of thumb from http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html "Capitalize all words in titles of publications and documents, except a, an, the, at, by, for, in, of, on, to, up, and, as, but, or, and nor."

**Comment by edenhill** [comment]
LGTM

**Pull request merged**